src/main/java/Duke.java

```diff
@@ -74,9 +75,28 @@ public void run() {
                         tokens = parser.getDescription().split(" /at ");
                         ui.transmitMessage("Got it. I've added this task:");
                         this.tasks.add(new Events(tokens[0], tokens[1]));
-
+                        break;
+                    case "find":
```

Comment:
Perhaps you can create another class Find that extends Task to handle the actions